src/routes/executions.js

```diff
@@ -0,0 +1,46 @@
+import { Router } from "express"
+import { Container } from "typedi"
+import { Auth, ExecutionService } from "../services"
+import config from "../config"
+
+export default appRouter => {
+  const router = Router()
+  const authService = Container.get(Auth)
+
+  //Validação do Codigo de Entrada
+  router.post("/", authService.middlewares.requireAppAdministration, (req, res) => {
```

Comment:
Duas coisas:
- tens de mudar o nível de autenticação para `autenticated`, senão apenas admins poderão fazer e requisição (erro meu em não te avisar pra mudar quando tu tava fazendo, sorry);
- a _arrow function_ tem que ser `async`, porque você precisa esperar salvar no banco (operação assíncrona) pra retornar pro cliente (deixei um comentário na linha 42 também).

```suggestion
  router.post("/", authService.middlewares.authenticated, async (req, res) => {
```